src/sensr_message_listener.cpp

```diff
@@ -20,7 +20,7 @@ namespace sensr {
     }
 
     void MessageListener::OnGetOutpuMessage(const sensr_proto::OutputMessage &message) {
-        std::cout << "Deprecated Message" << std::endl;
+        //std::cout << "Deprecated Message" << std::endl;
```

Comment:
Why did you comment this out?

Reply:
Ah this message kept showing because no class in console_sample.cpp override this function.

Reply:
then is that meaning we keep calling GetOutpuMesage function somewhere in console_sample.cpp? I wonder how that function is called because we already replace typos.

Reply:
OnGetOutpuMessage is called in sensr_client internally. If console_sample does not override OnGetOutpuMessage(), then the default function(MessageListener::OnGetOutpuMessage) will be called. And the reason we keep GetOutpuMessage is to provide user to prepare time to replace OnGetOutpuMessage to OnGetOutputMessage.
OnGetOutpuMessage will be removed after 1 cycle release.

Reply:
Ah, I see. Thanks for the explanation!